main.go

```diff
@@ -6,6 +6,10 @@ import (
 	"github.com/joyent/containerpilot/core"
 
 	log "github.com/Sirupsen/logrus"
+
+	// Import backends so that they initialize
```

Comment:
There's a bit of a code smell to me with this -- the individual backends are getting imported all the way at the top of our application which is leaking a lot of implementation detail upwards. Is it possible to push this import down into the `discovery` package?